kubetest/aks.go

```diff
@@ -0,0 +1,297 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
is the boilerplate header correct here?

Reply:
this actually came up in cluster-api -- I don't know if we're supposed to update these or not?

Reply:
s/2018/2020/

Reply:
fixed